core/logging_api.php

```diff
@@ -125,11 +126,18 @@ function log_event( $p_level, $p_msg ) {
 
 	$t_php_event = $t_now . ' ' . $t_level . ' ' . $t_msg;
 
+
 	switch( $t_destination ) {
 		case 'none':
 			break;
 		case 'file':
-			error_log( $t_php_event . PHP_EOL, 3, $t_modifiers );
+			if( $s_check_log_writable && isset( $t_modifiers ) && !( is_writable ( $t_modifiers ) ) ){
+				trigger_error( sprintf( lang_get('warning_log_not_writable'), $t_modifiers ) );
+				$s_check_log_writable = false;	
+			}
+			if( isset( $t_modifiers ) && ( is_writable ( $t_modifiers ) ) ){
```

Comment:
It would be good if we avoid calling is_writable() twice for the first check case.